copy/Cargo.toml

```diff
@@ -17,7 +17,8 @@ thiserror = "1.0"
 tracing = "0.1"
 
 [dependencies.git2]
-version = ">= 0.13.12, 0.13"
+git = "https://github.com/radicle-dev/git2-rs.git"
```

Comment:
It might be cleaner to add a patch to the workspace toml and leave these files as-is